components/financial_assistance/app/controllers/financial_assistance/applications_controller.rb

```diff
@@ -22,7 +22,11 @@ class ApplicationsController < FinancialAssistance::ApplicationController
     # We should ONLY be getting applications that are associated with PrimaryFamily of Current Person.
     # DO NOT include applications from other families.
     def index
-      @applications = FinancialAssistance::Application.where("family_id" => get_current_person.financial_assistance_identifier)
+      @applications = FinancialAssistance::Application.where("family_id" => get_current_person.financial_assistance_identifier).desc(:created_at)
```

Comment:
this change is not feature flagged, it needs to be because it will change the default behavior on DC